### PR TITLE
aui select and checkbox added

### PR DESCRIPTION
**Pull request by kkrisstoff**

### CODE REVIEW

**Comment by yanas** [comment]
screenshot 2016-10-17 23 52 38

Would it be possible to make the scroll background the same as the background (change it from black to grey). It looks strange this way.

File: modules/UI/side_pannels/settings/SettingsMenu.js [hunk 1]
```diff
@@ -61,43 +62,73 @@ function generateDevicesOptions(items, selectedId, permissionGranted) {
     return options.join('');
 }
 
+function initSelect2($el, onSelectedCb) {
```
Comment (yanas) [inline modules/UI/side_pannels/settings/SettingsMenu.js]: Jsdoc missing.

**Comment by yanas** [comment]
While we're changing the dropdowns in the Settings container, shouldn't we also change the two text fields in the Profile container? This can of course be done in a separate PR. What worries me is that the font looks very different.

**Comment by kkrisstoff** [comment]
I changed styles. But don't think it looks better now..
screen shot 2016-10-18 at 6 01 15 pm
screen shot 2016-10-18 at 6 01 33 pm

**Comment by kkrisstoff** [comment]
> While we're changing the dropdowns in the Settings container, shouldn't we also change the two text fields in the Profile container? This can of course be done in a separate PR. What worries me is that the font looks very different.

We have a special class for inputs. Will add in in all inputs But should do it in a separate PR

File: css/aui-components/dropdown.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+.select2-results{
+    &::-webkit-scrollbar {
+        background-color: transparent;
+    }
+    &::-webkit-scrollbar-track {
+        background-color: transparent;
+    }
+    &::-webkit-scrollbar-track-piece {
+        background-color: transparent;
+    }
```
Comment (lyubomir) [inline css/aui-components/dropdown.scss]: @kkrisstoff, why are `&::-webkit-scrollbar`, `&::-webkit-scrollbar-track`, and `&::-webkit-scrollbar-track-piece` the same i.e. `background-color: transparent;` and yet written separately? Is that out of a convenience for future editing should there arise a need to have them defined differently?

File: css/_side_toolbar_container.scss [hunk 3]
```diff
@@ -72,16 +74,16 @@
...
+            margin: 10px 0;
         }
 
         /**
          * Main title size.
          */
-        > div.title {
+        div.title {
             color: $defaultColor !important;
             text-align: center;
```
Comment (kkrisstoff) [inline css/_side_toolbar_container.scss]: Yep. It overwrites stiles above. I don't think it discord. I can rewrite it separately and add one more rule for `div.subTitle `

**Comment by lyubomir** [comment]
@kkrisstoff, I see in the Console of Chrome's Developer Tools a request for select2x2.png results in a 404 response. The file select2x2.png is referenced by AUI but we're not deploying it. I and Yana don't see that we're really using the file. Anyway, it's undesirable to have a request that results in a 404 response.

File: css/_side_toolbar_container.scss (hunk 3 above)
Comment (lyubomir) [inline css/_side_toolbar_container.scss]: You have separate rules for `div.title` and `div.subTitle` already.

**Pull request merged**